### DISCUSSION TITLE
how to make watermark non moveable?

### DISCUSSION THREAD

**Discussion by xomarko** [danielkrupinski/Osiris#3864] [Q&A]
help pls

**Comment by Xsintashi** [comment]
Add `  | ImGuiWindowFlags_NoMove` [Here](https://github.com/danielkrupinski/Osiris/blob/master/Source/Hacks/Misc.cpp#L404). Should work if pasted properly

**Reply by xomarko** [reply]
> Add ` | ImGuiWindowFlags_NoMove` [Here](https://github.com/danielkrupinski/Osiris/blob/master/Source/Hacks/Misc.cpp#L404). Should work if pasted properly

how can i make it positioned to top right?

**Reply by Xsintashi** [reply]
mess with `ImGui::SetNextWindowPos()` function like 

`float screenX = ImGui::GetIO().DisplaySize.x;`
`ImGui::SetNextWindowPos({ screenX - 128.f , 96.f});`

Not sure will it work. I wrote this in notepad, without check in game but try also mess with 128.f number, decrease/increase whatever.
